crypto/types.go

```diff
@@ -100,3 +100,19 @@ const (
 
 // Signature is a generic type, regardless of the signature scheme
 type Signature []byte
+
+// InvalidInputs is an error returned when a crypto API receives invalid inputs.
+// It allows a function caller differentiate unexpected program errors from errors caused by
+// invalid inputs.
+type InvalidInputs struct {
```

Comment:
Should this type name have a suffix of `Error`?

Reply:
I think you're right. The examples I looked at all have the suffix.